.env.test

```diff
@@ -14,4 +14,7 @@ TOKEN=
 
 TODO_IDLIST=Env_Test_ToDo_ID
 DOING_IDLIST=Env_Test_Doing_ID
-DONE_IDLIST=Env_Test_Done_ID
\ No newline at end of file
+DONE_IDLIST=Env_Test_Done_ID
+
+MONGO_DB_CONNECTION_STRING=mongodb://fakemongo.com
+MONGO_DB_NAME=mockdb01
```

Comment:
We should now remove all the Trello variables above since they're unused, and it would be good to make similar changes to the `.env.template` file too so that new users can easily find what they need to set

Reply:
Cheers Jack, will remove any reference to these Trello variables from .env.test + .env.template